.github/workflows/cypress.yml

```diff
@@ -0,0 +1,143 @@
+name: Cypress
+
+on: [push]
+
+env:
+  APP_NAME: richdocuments
+  CYPRESS_baseUrl: http://172.17.0.1:8081/index.php
+  CYPRESS_collaboraUrl: http://172.17.0.1:9980
+
+jobs:
+  cypress:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14.x]
+        containers: [1, 2, 3, 4]
+        php-versions: [ '7.4' ]
+        databases: [ 'sqlite' ]
+        server-versions: [ 'master' ]
+
+    name: runner ${{ matrix.containers }}
+
+    services:
+      collabora:
+        image: collabora/code:21.11.2.4.1
```

Comment:
We need to handle the welcome popup separately with the follow up release during testing.